src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,41 @@
+package seedu.addressbook.data.tag;
+
+import seedu.addressbook.data.person.*;
+
+public class Tagging {
+	
+	private Person person;
+	private Tag tag;
+	private boolean isAdded; // true if its adding a tag, false if its deleting a tag
```

Comment:
Boolean is better than using string, but you might like to look into using Enums. `isAdded = false` might imply not added semantically, instead of deleting.